### PR TITLE
Version: Changes version to dev9999

### PR DESCRIPTION
**Pull request by gopalmeena**

### CODE REVIEW

**Comment by sudheesh001** [comment]
@gopalmeena Welcome to coala and thanks for making your first pull request to coala.

We have `gitmate-bot` which helps us finding out a few problems before the review process can proceed. You need to run the coala command after you make a commit, In this case, the commit message is actually too long. Here's the documentation on how to make good commit messages.

http://coala.readthedocs.org/en/latest/Getting_Involved/Writing_Good_Commits.html

File: coalib/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.4.2.dev0
+0.4.2.dev9999
```
Comment (sudheesh001) [inline coalib/VERSION]: Add new line

**Comment by sudheesh001** [comment]
unack 95294ff

**Comment by sudheesh001** [comment]
@gopalmeena This is good, but we need to add more information to the commit message telling why this change was made, the answer to that is in the issue #1800 

> when we use the git version and try installing things, it gets messed up because the git has dev0 which is lower than the current nightly release. This is conceptually wrong as the latest thing in git is the latest most beta version of coala.

rephrase this and amend the git long message so that we can merge this.

**Comment by sils1297** [comment]
hey @gopalmeena if you need help please tell us! :) You can also reach us on https://gitter.im/coala-analyzer/coala

**Comment by gopalmeena** [comment]
Yeah sure, if i need any help , i will join gitter. Thanks for help @sils1297 and @sudheesh001 for mentoring

**Comment by AbdealiJK** [comment]
Hey can we do this quick ?
This keeps coming up in lots of places and now is importance/high IMO

**Comment by gopalmeena** [comment]
Ok. I will fix it soon.

**Comment by sudheesh001** [comment]
Good work @gopalmeena 

@AbdealiJK I request you to review too.

**Comment by sudheesh001** [comment]
ack 304c8e0

**Comment by hypothesist** [comment]
@gopalmeena Could you edit the PR description too? It could just be the commit message you used.

**Comment by AbdealiJK** [comment]
Looks good to me. restarted appveyor as i dont know why it failed

**Comment by hypothesist** [comment]
failing at `self.assertEqual(Lint.check_prerequisites(), Lint.prerequisite_fail_msg)`: could it have something to do with the prerequisites failing due to version change?

**Comment by AbdealiJK** [comment]
@gopalmeena there is something weird going on.
I'm not sure why appveyor fails.
We're discussing this o gitter if you want to join

**Comment by AbdealiJK** [comment]
I've moved your commit to https://github.com/coala-analyzer/coala/pull/1878
So that I can fix the appveyor issues that have come up (Which aren't your fault.)
You are still the author of the commit (even in that PR)

I am leaving this PR open (in pending review), as I still have no clue why appveyor starting showing these errors now of all times. It should have shown these errors earlier.

@sils1297 @Makman2 ideas ?

**Pull request closed without merging**